packages/absolute/absolute.0.1/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: "Marie Pelleau <marie.pelleau@i3s.unice.fr>"
+authors: [
+  "Marie Pelleau <marie.pelleau@i3s.unice.fr>"
+  "Ghiles Ziat <ghiles.ziat@lip6.fr>"
+]
+homepage: "https://github.com/mpelleau/AbSolute"
+bug-reports: "https://github.com/mpelleau/AbSolute/issues"
+dev-repo: "git+https://github.com/mpelleau/AbSolute"
+license: "LGPL"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.1"}
```
For forward compatibility reasons, dune cannot be tagged with build.
According to your dune-project file, you need at least dune 1.1.

Can you also upstream those changes? (no need to re-release)